codalab/lib/bundle_cli.py

```diff
@@ -176,8 +176,8 @@ def print_help(self, out_file=None):
     def error(self, message):
         # Adapted from original:
         # https://hg.python.org/cpython/file/2.7/Lib/argparse.py
-        if len(sys.argv) == 1:
-            self.print_help()
+        if len(sys.argv) == 1 or (len(sys.argv) == 2 and sys.argv[1] in ['--help', '-h']):
```

Comment:
Oh, I must've branched off of my feature branch rather than master. I'll just remove them manually.